core/runtime/src/main/java/io/quarkus/runtime/configuration/ConfigRecorder.java

```diff
@@ -1,5 +1,6 @@
 package io.quarkus.runtime.configuration;
 
+import static io.quarkus.runtime.ConfigConfig.*;
```

Comment:
We don't want the star imports.

Reply:
Strange. I thought the `impsort` plugin would get rid of it.

Reply:
Unfortunately, no, it doesn't handle star imports.

Reply:
BTW, have you tried making the process as slow as possible to reproduce it? Either with `nice` or by lowering your CPU frequency?

Experience shows that CI slowness usually make concurrency issues pop up.

Reply:
I did not... I'll try it.